src/data_loader/population/__init__.py

```diff
@@ -53,3 +29,89 @@ def get_origin_points(
 
     # Convert the origin points to a list of strings
     return list(origin_points["id"])
+
+
+def population_data_from_geojson(file_name: str) -> gpd.GeoDataFrame:
+    """
+    Loads a GeoJSON file and returns a GeoDataFrame.
+    :param file_name: The name of the population GeoJSON file.
+    :return: A GeoDataFrame containing the data from the GeoJSON file.
+    """
+    try:
+        population = gpd.read_file(POPULATION_DIR / file_name)
+        return population
+
+    except Exception as e:
+        logging.error(f"Error loading GeoJSON file: {e}")
+        exit(0)
```

Comment:
Exit code 0 indicates success. Would probably want to raise an error instead
```suggestion
        exit(1)
```